### ISSUE TITLE
Bug: style element indents

### ISSUE THREAD

**Issue by what1s1ove** [prettier/plugin-pug#278]
## Info

| Tool      | Version                 |
| --------- | ----------------------- |
| Plugin    | v1.16.2                  |
| Prettier  | v2.3.1                  |
| Framework | none |
| Node      | v14.17.3                  |
| OS        | mac           |

## Prettier config

```yml
tabWidth: 4
bracketSpacing: true
printWidth: 120
singleQuote: true
trailingComma: none
useTabs: true
semi: true
arrowParens: avoid
endOfLine: auto
overrides:
  - files: '*.pug'
    options:
      pugSingleQuote: 'true'
      pugAttributeSeparator: always
      pugClosingBracketPosition: new-line
      pugWrapAttributesThreshold: 1
      pugExplicitDiv: false
      pugClassNotation: literal
      pugEmptyAttributes: none
      pugSortAttributes: asc
      pugSortAttributesBeginning:
        - id
        - class
        - src
        - href
        - type
        - width
        - height
        - alt
        - aria-
        - data-
```

## Input

```pug
// one
html
	head
		style(type='text/css').
			.card-body {
				background-color: white;
				color: #2c2c2c;
				font-size: 0.9rem;
				padding: 25px;
			}		// additional indent
		block styles
	body

// two
block content
	style(type='text/css').
		.entry__name {
			text-align: center;
			margin: 0 0 60px;
		}	// additional indent
	h1.entry__title
		b Credentialing Packet

// three
block content
	style(type='text/css').
		.entry__generalinfo-list {
			display: -webkit-box;
			display: -webkit-flex;
			display: flex;
			-webkit-flex-wrap: wrap;
			flex-wrap: wrap;
		}
	// additional indent
	h2.title General Information
```

## Expected Output

```pug
// one
html
	head
		style(type='text/css').
			.card-body {
				background-color: white;
				color: #2c2c2c;
				font-size: 0.9rem;
				padding: 25px;
			}
		block styles
	body

// two
block content
	style(type='text/css').
		.entry__name {
			text-align: center;
			margin: 0 0 60px;
		}
	h1.entry__title
		b Credentialing Packet

// three
block content
	style(type='text/css').
		.entry__generalinfo-list {
			display: -webkit-box;
			display: -webkit-flex;
			display: flex;
			-webkit-flex-wrap: wrap;
			flex-wrap: wrap;
		}

	h2.title General Information
```

## Additional Context

Hi there!

Several problems with indents in the `style` element.

I hope this helps make your package even better. Thank you 🙏

**Comment by Shinigami92** [comment]
@what1s1ove Thanks for being part of my community :slightly_smiling_face: 
I released [v1.16.4](https://www.npmjs.com/package/@prettier/plugin-pug/v/1.16.4) for you

**Comment by what1s1ove** [comment]
Thaaanks @Shinigami92, you're beautiful 😻
